src/redux/gas.js

```diff
@@ -115,7 +115,9 @@ export const gasPricesStartPolling = () => async (dispatch, getState) => {
           // Add gas estimates
           adjustedGasPrices = await etherscanGetGasEstimates(priceData);
         } catch (e) {
-          logger.log('falling back to eth gas station', e);
+          captureException(new Error('Etherscan gas estimates failed'));
+          logger.log('Etherscan gas estimates error:', e);
```

Comment:
did you mean sentry.log for some of these

Reply:
logger.sentry! 😂🙌🏻